compat/inet_ntop.cc

```diff
@@ -150,37 +150,33 @@ inet_ntop4(const u_char *src, char *dst, size_t size)
 /* const char *
  * inet_ntop6(src, dst, size)
  *  convert IPv6 binary address into presentation (printable) format
- * author:
+ * original author:
  *  Paul Vixie, 1996.
  */
 static const char *
 inet_ntop6(const u_char *src, char *dst, size_t size)
 {
     /*
      * Note that int32_t and int16_t need only be "at least" large enough
-     * to contain a value of the specified size.  On some systems, like
-     * Crays, there is no such thing as an integer variable with 16 bits.
+     * to contain a value of the specified size.  On some systems
+     * there is no such thing as an integer variable with 16 bits.
```

Comment:
IMO, drop this change. The text is still correct for this code's implementation design choices even if not relevant to Squid in general.

Reply:
The comment is wrong or extremely misleading (in its current context), but I agree that just removing Crays does more harm than good and support the recommendation to keep the official comment "as is" in this PR.

Reply:
reverting